src/authentication/loginWithCustomTokenCommand.ts

```diff
@@ -0,0 +1,12 @@
+import { window } from "vscode";
+import { loginWithCustomToken } from "./authentication.api";
+
+export default async function loginWithCustomTokenCommand() {
+  const customToken = await window.showInputBox({
+    prompt: "Enter your auth token",
+    ignoreFocusOut: true,
+  });
+  if (customToken) {
```

Comment:
is there a need to validate the input in some way? or worse case nothing will happen?
What happens if we supply a corrupted custom token?